test/testSetupFile.js

```diff
@@ -0,0 +1 @@
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 70000;
```

Comment:
Is there a reason why this is set to `70000`? If I understand it correctly, previously in Mocha it was set to `10000`.